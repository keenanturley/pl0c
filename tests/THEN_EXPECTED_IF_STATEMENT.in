var x, w;
begin
   x:= 4;
   w:= 7;
   if w > x
	w:= w + 1;
	w:= x; 
end.