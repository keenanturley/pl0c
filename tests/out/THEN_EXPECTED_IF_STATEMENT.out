Lexeme List:
29 2 x 17 2 w 18 21 2 x 20 3 4 18 2 w 20 3 7 18 23 2 w 13 2 x 2 w 20 2 w 4 3 1 18 2 w 20 2 x 18 22 19 

Lexeme List (Symbolic Representation):
varsym identsym x commasym identsym w semicolonsym beginsym identsym x becomessym numbersym 4 semicolonsym identsym w becomessym numbersym 7 semicolonsym ifsym identsym w gtrsym identsym x identsym w becomessym identsym w plussym numbersym 1 semicolonsym identsym w becomessym identsym x semicolonsym endsym periodsym 

Error: Then keyword expected following if statement.
